classroom.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'student'
+
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
Great job @atatm 🤝

- [ ] I suggest you implement this method in the `Student` class instead of the `Classroom` class since it creates a student belongs to a classroom relationship.
- [ ] Could you make sure the classroom class implements a 'has-many' relationship? For example,  `a classroom has many students`
![add_student](https://user-images.githubusercontent.com/63917725/224483883-38b8125c-a361-4575-b483-930677cc78ce.PNG)